azerothcore/azerothcore-wotlk
# fix(Scripts/Temple of Ahn Qiraj): Skeram should cast True Fulfillment…

… on closest non-tank player.

Fixes #12713



## Issues Addressed:

- Closes #12713

## Tests Performed:

- Not tested.

## How to Test the Changes:

`.go xyz -8295.55 2102.6 118.2 531 3.6` with 1 character
engage boss, see if True Fulfillment is being cast
`.go xyz -8295.55 2102.6 118.2 531 3.6` with 2 characters
see if the player closest to the boss is mind controlled


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
